docs: desktop build instructions (#53)
mrs-electronics-inc/mrs-sdk-qt: docs/src/content/docs/get-started/configure-project.mdx

---
title: Configure a Project
description: This page explains how to configure a Qt project to use the MRS SDK.
sidebar:
  order: 3
---

import { Aside, Tabs, TabItem } from '@astrojs/starlight/components';

Qt projects using the SDK need some additional configuration to pull in and use the SDK's build system, which takes care of defining a consistent build-kit identity and linking the correct compiled library for the project's specified device/OS target.

<Aside type="caution">
Trying to configure a project with an incorrect build kit will cause errors that prevent compilation.

> Make sure you have followed all steps for [setting up build kits](/get-started/set-up-build-kits) before trying to configure a project.

Reviewer: ```suggestion
Make sure you have followed all the steps for [setting up build kits](/get-started/set-up-build-kits) before trying to configure a project.
```

</Aside>

## Steps

### Set the consumer target

The first step in setting up a project to use the Qt SDK is to tell the SDK's build system the name of the target that will consume the SDK.

<Tabs syncKey="buildtool">
    <TabItem label="CMake">

>         For most projects, this will be equal to the name of the target declared in the `project` command:

Reviewer: ```suggestion
        For most projects, this will be the same as the name of the target declared in the `project` command:
```
